.github/workflows/main.yml

```diff
@@ -0,0 +1,15 @@
+name: Dependabot Auto Approve
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  auto_approve:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: "hmarr/auto-approve-action@v1.0.0"
```

Comment:
After approving a few of these, I see this @v1.0.0 over and over.. I presume there's no easy way to inject the version or "uses" line?

Reply:
I don't think there's a way to inject it, but I should probably update some of these to 2.0.0, as that's the latest currently.